src/api/firebase.js

```diff
@@ -72,11 +76,41 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 		totalPurchases: 0,
 	});
 }
+/**
+ * Update an existing item in the user's list in Firestore.
+ * @param {string} listId The id of the list we're adding to.
+ * @param {Object} item Information about the new item.
+ **/
+export async function updateItem(listId, item) {
+	const {
+		id,
+		dateCreated,
+		dateLastPurchased,
+		dateNextPurchased,
+		totalPurchases,
+	} = item;
+
+	const transformToJSDate = (date) => {
```

Comment:
This is a helper function takes in a Firestore Date object and returns a JS Date object that can be used.

Reply:
It feels really useful to turn this into a helper function so it can be reused.